server-extension/src/main/java/flyway/paikkis/V2_36__use_wfs_vector_layer_plugin.java

```diff
@@ -0,0 +1,57 @@
+package flyway.paikkis;
+
+import fi.nls.oskari.domain.map.view.Bundle;
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+import fi.nls.oskari.map.view.AppSetupServiceMybatisImpl;
+import fi.nls.oskari.map.view.ViewService;
+import fi.nls.oskari.util.FlywayHelper;
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.sql.Connection;
+import java.util.List;
+
+/**
+ * Replaces transport based wfs plugin with vector implementation
+ */
+public class V2_36__use_wfs_vector_layer_plugin implements JdbcMigration {
+
+    private static final Logger LOG = LogFactory.getLogger(V2_36__use_wfs_vector_layer_plugin.class);
+    private static final String BUNDLE_NAME = "mapfull";
+    private static final String WFS_TRANSPORT_PLUGIN_ID = "Oskari.mapframework.bundle.mapwfs2.plugin.WfsLayerPlugin";
+    private static final String WFS_VECTOR_PLUGIN_ID = "Oskari.wfsvector.WfsVectorLayerPlugin";
+
+    private ViewService viewService;
+
+    public void migrate(Connection connection) throws Exception {
+        viewService = new AppSetupServiceMybatisImpl();
+        List<Long> viewIds = FlywayHelper.getViewIdsForTypes(connection, "DEFAULT", "USER", "PUBLISHED");
```

Comment:
Looks like you can omit the types and just pass connection to get all the views.

Reply:
Refactored as suggested.